Client/trunk/externals/assimp-4.0.0/code/FBXDocument.h

```diff
@@ -667,7 +667,7 @@ class Video : public Object
         }
```

Comment:
https://github.com/assimp/assimp/blob/a2b8d66a86e3e16d15b55fbd4d7f8f164e297131/code/FBXDocument.h#L661

this is code added by us?  because the original implementation has no memcpy.

Reply:
@kkvskkkk

Reply:
assimp在2个材质使用同一个纹理贴图并且是内嵌图片数据时会崩溃。 这个代码是修复这个问题而添加的， 之前把指针设置为nullptr是为了跟以前的语义保持一致性。 不设置为nullptr不会影响代码逻辑。